src/Nervos/CellOutput.h

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Script.h"
+#include "../Data.h"
+#include "../proto/Nervos.pb.h"
+
+#include <memory>
+
+namespace TW::Nervos {
+
+/// Nervos cell output.
+struct CellOutput {
+    int64_t capacity;
+    Script lock;
+    Script type;
+
+    /// Initializes an empty cell output.
+    CellOutput() = default;
+
+    /// Initializes a cell output with a capacity and scripts.
+    CellOutput(int64_t capacity, Script lock, Script type)
+        : capacity(capacity), lock(lock), type(type) {}
```

Comment:
consider using `std::move` to avoid unnecessary copy on lock and type

Reply:
Done